modules/openapi-generator/src/main/resources/Javascript-Apollo/api_test.mustache

```diff
@@ -1,13 +1,15 @@
 {{>licenseInfo}}
 // CommonJS-like environments that support module.exports, like Node.
-factory(require('expect.js'), require(process.cwd()+'/src/{{#invokerPackage}}{{.}}/{{/invokerPackage}}index'));
+//factory(require('expect.js'), require(process.cwd()+'/src/{{#invokerPackage}}{{.}}/{{/invokerPackage}}index'));
```

Comment:
I have no idea what this does. `factory` does not exist anywhere.

The second `require` is the inclusion of the Api being tested in this file, which is included below in a way that works (and is more human-readable/writeable)

Reply:
I think these are carried over or copied from https://github.com/OpenAPITools/openapi-generator/blob/5d68bd6a03f0c48e838b4fe3b98b7e30858c0373/modules/openapi-generator/src/main/resources/Javascript/es6/model_test.mustache#L8, in which this generator referenced.